All frontpage builders are failing on coverage report

```
Traceback (most recent call last):
  File "create_test_coverage_report.py", line 52, in <module>
    "ERROR: UNIT TESTS FAILED, PLEASE FIX BEFORE RUNNING COVERAGE:\n%s\n%s" % (output_stream.getvalue(), ''.join([repr(error) for error in reporter.errors]))
AssertionError: ERROR: UNIT TESTS FAILED, PLEASE FIX BEFORE RUNNING COVERAGE:
ipv8.test.REST.test_attestation_endpoint.TestAttestationEndpoin ... [ERROR]
```

--- a/test_classes_list.txt
+++ b/test_classes_list.txt
@@ -42,3 +42,4 @@ ipv8/test/dht/test_routing.py:TestRoutingTable
 ipv8/test/dht/test_storage.py:TestStorage
 
 ipv8/test/REST/test_attestation_endpoint.py:TestAttestationEndpoint
+

